### PR TITLE
Show more per-camera telemetry on SmartDashboard

### PR DESCRIPTION
**Pull request by ivanggq**
- show confidence multiplier and estimated pose per camera on SmartDashboard
- shortened and organized the vision-related telemetry in the format `Vision/<camera name> - <description of data>`
- removed unused imports

### CODE REVIEW

File: src/main/java/frc/robot/Robot.java
Comment (ivanggq) [inline src/main/java/frc/robot/Robot.java]: Removed a bunch of unused imports to reduce the issues shown in VS Code.

File: src/main/java/frc/robot/subsystems/vision/Vision.java
Comment (ivanggq) [inline src/main/java/frc/robot/subsystems/vision/Vision.java:27]: Exposed the camera name from the `Vision` class.

File: src/main/java/swervelib/motors/SparkFlexSwerve.java
Comment (ivanggq) [inline src/main/java/swervelib/motors/SparkFlexSwerve.java]: Didn't want to delete this unused guys in case somebody is looking for them...

File: src/main/java/frc/robot/subsystems/vision/VisionEstimation.java
Comment (ivanggq) [inline src/main/java/frc/robot/subsystems/vision/VisionEstimation.java]: Reporting the pose from each camera separately so we can see what each camera thinks.

File: src/main/java/frc/robot/subsystems/vision/VisionEstimation.java
Comment (ivanggq) [inline src/main/java/frc/robot/subsystems/vision/VisionEstimation.java]: Adding the current confidence multiplier to this `MeanPerPeriodTelemetry` helper, which will send the mean (avg) conf. mult. every half sec to SmartDashboard.

File: src/main/java/frc/robot/subsystems/vision/VisionEstimation.java
Comment (ivanggq) [inline src/main/java/frc/robot/subsystems/vision/VisionEstimation.java]: The `periodic()` function of these helper telemetry classes will send the data to SmartDashboard once every sec or half sec (the reporting period is specified in their c'tor on lines 89-101).

File: src/main/java/frc/robot/subsystems/vision/Vision.java
Comment (Boomaa23) [inline src/main/java/frc/robot/subsystems/vision/Vision.java]: Could use `String.format()` here if you want; not required though

File: src/main/java/frc/robot/subsystems/vision/Vision.java
Comment (Boomaa23) [inline src/main/java/frc/robot/subsystems/vision/Vision.java]: Is there a reason these are here but commented? Remove if no reason

File: src/main/java/frc/robot/subsystems/vision/VisionEstimation.java
Comment (Boomaa23) [inline src/main/java/frc/robot/subsystems/vision/VisionEstimation.java:1]: lots of commented things in this file. if you intend to use them move to another more relevant place, otherwise remove

File: src/main/java/swervelib/motors/SparkMaxSwerve.java
Comment (Boomaa23) [inline src/main/java/swervelib/motors/SparkMaxSwerve.java]: wut dis

**Review by Boomaa23 — DISMISSED** [review]
looks good to me content wise! again just minor refactoring, but honestly it'd be fine as is

File: src/main/java/frc/robot/subsystems/vision/Vision.java
Comment (ivanggq) [reply src/main/java/frc/robot/subsystems/vision/Vision.java]: I think I will create a simple helper function that returns this telemetry string as this string/pattern is currently duplicated on 8+ places (vision/ + cameraname + telemetry name).

File: src/main/java/frc/robot/subsystems/vision/Vision.java
Comment (ivanggq) [reply src/main/java/frc/robot/subsystems/vision/Vision.java]: Will remove. This was some code from @aayushext , not sure if we planned to use it.

File: src/main/java/frc/robot/subsystems/vision/VisionEstimation.java
Comment (ivanggq) [reply src/main/java/frc/robot/subsystems/vision/VisionEstimation.java:1]: The telemetry lines/variables are usually in sets of 4 - one per camera. But since at the moment of this PR we had only 2 cameras, we added 2 uncommented lines and 2 commented lines. Eventually, all 4 cameras will be added and all code should be uncommented.

File: src/main/java/swervelib/motors/SparkMaxSwerve.java
Comment (ivanggq) [reply src/main/java/swervelib/motors/SparkMaxSwerve.java]: Commented out unused variable, was unsure if it has sentimental value to the author who added it :) Might remove it now...

**Comment by ivanggq** [comment]
In the interest of time and in preparation for testing vision code today, I am going to merge this PR, dismissing your review @Boomaa23 . I think I addressed your comments but please let me know if you think more changes are needed and we can make changes in a different PR. Thanks.

**Comment by ivanggq** [comment]
Hm, I cannot merge anyway without an approval, so will wait for an approval :) @Boomaa23 @AlexanderGardiner @aayushext @Coolkid0629

**Pull request merged**
